Makefile

```diff
@@ -31,8 +31,9 @@ test-efm:
 
 test-syntax:
 	$(SILENT) vim -T dumb -E \
+	        -c 'source test/functions.vim' \
 		-c 'source test/run_test.vim' \
-		-c 'call RunTestSyntax()' > test-syntax.log
+		-c 'call RunTestSyntax()' | tr -d '[
]' > test-syntax.log
```

Comment:
The file was appearing "broken" in my system, like it had been created in DOS, so I just stripped the extra carriage-return signals. Could I ask you to remove it and see if it works without it in your system?